cmd/main.go

```diff
@@ -157,8 +155,9 @@ func main() {
 		TestProfilesPersister:  testConfigPersister,
 		GenericPersister:       dbHandler,
 		GraphqlHandler: graphql.New(graphql.Options{
-			DBHandler:  &dbHandler,
-			KubeClient: &mesheryKubeClient,
+			DBHandler:        &dbHandler,
+			GetKubeClient:    helpers.NewKubeClientGenerator(viper.GetString("KUBECONFIG_FOLDER")),
+			GetDynamicClient: helpers.NewDynamicClientGenerator(viper.GetString("KUBECONFIG_FOLDER")),
```

Comment:
Post the conference, lets work on getting all of these clients as a part of the meshery-kube-client from meshkit @utkarsh-pro